x-pack/plugins/secops/public/components/range_date_picker/quick_select_popover/commonly_used.tsx

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGrid, EuiFlexItem, EuiLink, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
+import { getOr } from 'lodash/fp';
+import moment, { Moment } from 'moment';
+import React from 'react';
+import { pure } from 'recompose';
+import { DateType, RecentlyUsedI } from '..';
+
+export enum DatePickerOptions {
+  today = 'today',
+  yesterday = 'yesterday',
+  thisWeek = 'this-week',
+  weekToDate = 'week-to-date',
+  thisMonth = 'this-month',
+  monthToDate = 'month-to-date',
+  thisYear = 'this-year',
+  yearToDate = 'year-to-date',
+}
+
+const commonDates: Array<{ id: string; label: string }> = [
+  {
+    id: DatePickerOptions.today,
+    label: 'Today',
+  },
+  {
+    id: DatePickerOptions.yesterday,
+    label: 'Yesterday',
+  },
+  {
+    id: DatePickerOptions.thisWeek,
+    label: 'This week',
+  },
+  {
+    id: DatePickerOptions.weekToDate,
+    label: 'Week to date',
+  },
+  {
+    id: DatePickerOptions.thisMonth,
+    label: 'This month',
+  },
+  {
+    id: DatePickerOptions.monthToDate,
+    label: 'Month to date',
+  },
+  {
+    id: DatePickerOptions.thisYear,
+    label: 'This year',
+  },
+  {
+    id: DatePickerOptions.yearToDate,
+    label: 'Year to date',
+  },
+];
+
+interface Props {
+  setRangeDatePicker: (from: Moment, to: Moment, type: DateType, msg: RecentlyUsedI) => void;
+}
+
+export const CommonlyUsed = pure<Props>(({ setRangeDatePicker }) => {
+  const links = commonDates.map(date => {
+    return (
+      <EuiFlexItem key={date.id}>
+        <EuiLink
+          onClick={updateRangeDatePickerByCommonUsed.bind(
```

Comment:
You can keep this `bind` if you want to pass arguments but since it returns a new reference this would be the same as just doing a

```ts
 <EuiLink onClick={() => updateRangeDatePickerByCommonUsed(date.id, setRangeDatePicker)}>
```

I would prefer the anonymous function at this point over mixing in bind's as a different pattern.